aws-py-serverless-raw/__main__.py

```diff
@@ -0,0 +1,160 @@
+"""Copyright 2016-2019, Pulumi Corporation.  All rights reserved."""
+import pulumi
+import pulumi_aws as aws
+import pulumi_aws.config
+from pulumi import Output
+import json
+
+# The location of the built dotnet3.1 application to deploy
+dotnet_application_publish_folder = "./app/bin/Debug/netcoreapp3.1/publish"
+dotnet_application_entry_point = "app::app.Functions::GetAsync"
+# The stage name to use for the API Gateway URL
+custom_stage_name = "api"
+
+#################
+## DynamoDB Table
+#################
+
+# A DynamoDB table with a single primary key
+counter_table = aws.dynamodb.Table("counterTable",
+                                   attributes=[
+                                       {
+                                           "name": "Id",
+                                           "type": "S",
+                                       },
+                                   ],
+                                   hash_key="Id",
+                                   read_capacity=1,
+                                   write_capacity=1,
+                                   )
```

Comment:
Well at first I think it might follow the Python convention for that. However, latter on I am thinking it might be clearly for the user to see if indentation is short since they don't need to read from right. So I would accept that